discovery-provider/plugins/notifications/src/server/routes/healthCheck.ts

```diff
@@ -0,0 +1,25 @@
+import { Router, Request, Response } from 'express'
+import { config } from '../../config'
+import { getRedisConnection } from '../../utils/redisConnection'
+
+// Mapped to the /health_check Endpoint
+const router = Router()
+
+router.get('/', async (req: Request, res: Response) => {
+  const redis = await getRedisConnection()
+  const lastIndexedMessageRedisKey = await redis.get(
+    config.lastIndexedMessageRedisKey
+  )
+  const lastIndexedReactionRedisKey = await redis.get(
+    config.lastIndexedReactionRedisKey
+  )
+  res.json({
```

Comment:
qq is this supposed to be returned or anything?

Reply:
nope, doesn't need to return anything